charts/example-configurations/minikube-values-gitlab.yaml

```diff
@@ -1,118 +1,47 @@
 # Configuration file for deploying the renku chart on minikube
-# against gitlab.com. We also configure the keycloak instance which is part
+# against gitlab.com. We also need to configure the keycloak instance which is part
 # of Renku to use gitlab.com as identity provider. See the developer docs
 # (https://renku.readthedocs.io/en/latest/developer/example-configurations/gitlab.html)
-# for instructions on how to use this values file .
+# for instructions on how to use this values file.
 
-ingress:
-  enabled: true
-  annotations:
-    kubernetes.io/ingress.class: 'nginx'
-    nginx.ingress.kubernetes.io/ssl-redirect: 'false'
-    nginx.ingress.kubernetes.io/proxy-body-size: '0'
-    nginx.ingress.kubernetes.io/proxy-request-buffering: 'off'
-  hosts:
-  - ""
+variables:
+  gitlabURL:          &gitlabURL           https://gitlab.com
+  gitlabClientId:     &gitlabClientId      # Put Application Id here!
+  gitlabClientSecret: &gitlabClientSecret  # Put Application Secret here!
+  notebooksRegistry:  &notebooksRegistry   registry.gitlab.com
```

Comment:
I'd rename this `registryHost`. Having "notebooks" in here might mislead people new to Renku into thinking that we have a special kind of registry just for notebooks.